reana_commons/email.py

```diff
@@ -30,10 +31,11 @@ def send_email(
     sender_email=REANA_EMAIL_SENDER,
 ):
     """Send emails from REANA platform."""
-    headers = "From: REANA platform <{from_}>\nTo: {to}\nSubject: {subject}".format(
-        from_=sender_email, to=receiver_email, subject=subject
-    )
-    message = "{headers}\n\n{body}".format(headers=headers, body=body)
+    message = EmailMessage()
+    message["From"] = f"REANA platform <{sender_email}>"
+    message["To"] = receiver_email
+    message["Subject"] = subject
```

Comment:
Do we need to set any header to specify message encoding?

Reply:
This should be already handled by [`set_content`](https://docs.python.org/3/library/email.contentmanager.html#email.contentmanager.set_content)